### PR TITLE
Fortinet fortigate enhancement

### PR DESCRIPTION
**Pull request by yasta5**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://jira-hq.paloaltonetworks.local/browse/CIAC-7488

## Description
This PR adds support for additional custom Fortinet Fortigate fields, and include enhancement for the readme.

### CODE REVIEW

File: Packs/FortiGate/README.md [hunk 1]
```diff
@@ -1,9 +1,26 @@
...
 
-In order to use the collector, you can use one of the following options to collect events from the vendor:
- - [Broker VM](#broker-vm)
+## Configuration on Server Side
+You need to configure Fortigate to forward Syslog messages.
 
-In either option, you will need to configure the vendor and product for this specific collector.
+1. Log in to the FortiGate web interface using your admin credentials.
+2. Open a CLI console by clicking the **_>** icon in the top right corner
+4. Run the below command:
```
Comment (ShirleyDenkberg) [inline Packs/FortiGate/README.md:11]:
```suggestion
4. Run the following command:
```

File: Packs/FortiGate/ReleaseNotes/1_0_24.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+#### Modeling Rules
+##### Fortinet FortiGate Modeling Rule
+Updated the modeling rule with additional fields.
```
Comment (ShirleyDenkberg) [inline Packs/FortiGate/ReleaseNotes/1_0_24.md:3]: Perhaps you should list the rules that were added?

**Comment by ShirleyDenkberg** [comment]
@yasta5 Doc review completed.

File: Packs/FortiGate/ReleaseNotes/1_0_24.md (hunk 2 above)
Comment (yasta5) [reply Packs/FortiGate/ReleaseNotes/1_0_24.md:3]: @ShirleyDenkberg 
I didn't add rules, but fields. It is a pretty long list to add for the release note so maybe should I rephrase it?

Comment (ShirleyDenkberg) [reply Packs/FortiGate/ReleaseNotes/1_0_24.md:3]: The description is clear.  I understand that you added field (my mistake in my comment). I guess you can leave it as it.

**Review by eepstain — APPROVED** [review]

**Pull request merged**